pyhf/readxml.py

```diff
@@ -235,14 +235,15 @@ def process_measurements(toplvl):
                 overall_param_obj['value'] = param.attrib['Val']
 
             # might be specifying multiple parameters in the same ParamSetting
-            for param_name in param.text.split(' '):
-                # lumi will always be the first parameter
-                if param_name == 'Lumi':
-                    result['config']['parameters'][0].update(overall_param_obj)
-                else:
-                    param_obj = {'name': param_name}
-                    param_obj.update(overall_param_obj)
-                    result['config']['parameters'].append(param_obj)
+            if param.text:
+                for param_name in param.text.split(' '):
+                    # lumi will always be the first parameter
+                    if param_name == 'Lumi':
+                        result['config']['parameters'][0].update(overall_param_obj)
```

Comment:
are we guaranteed that Lumi will always be at index 0?

Reply:
maybe we can add an `assert result['config']['parameters'][0]['name']=="Lumi"`?

Reply:
This wasn't code that I added.

Reply:
wasn't it added here? https://github.com/diana-hep/pyhf/pull/352 but granted not this PR

Reply:
Yeah, I guess it *was* code I added. I'll fix momentarily.

Reply:
This is not a problem. Lumi is always guaranteed to be at index 0 here.